packages/webpack/lib/configs/build.js

```diff
@@ -127,13 +127,8 @@ module.exports = function getConfig(config, name) {
       },
       minimizer: [
         new TerserPlugin({
-          cache: true,
-          parallel: true,
-          sourceMap: true,
```

Comment:
So `cache` and `parallel` are true by default but what about `sourceMap`?

Reply:
https://github.com/webpack-contrib/terser-webpack-plugin/pull/140